Gordon360/Services/DirectMessageService.cs

```diff
@@ -163,22 +163,6 @@ public List<Object> GetRoomById(string userId)
         public CreateGroupViewModel CreateGroup(String name, bool group, DateTime lastUpdated, string image, List<String> usernames)
         {
             DateTime createdAt = DateTime.Now;
-            Guid id = Guid.NewGuid();
-
-            var idParam = new SqlParameter("@_id", id.ToString());
-
-            var idCheck = RawSqlQuery<IdCheckViewModel>.query("CHECK_ID @_id", idParam);
-
-            foreach( IdCheckViewModel isNotAvailable in idCheck)
-            {
-                while (isNotAvailable.IdCheck)
-                {
-                    id = Guid.NewGuid();
-                    idParam = new SqlParameter("@_id", id.ToString());
-                    idCheck = RawSqlQuery<IdCheckViewModel>.query("CHECK_ID @_id", idParam);
-                }
-            }
-            var idRefreshParam = new SqlParameter("@_id", idParam.Value);
```

Comment:
Love to see code being refactored. It's nice to see that a foreach and while is not needed here. This makes the code a lot more simple.